gcp-observability/src/test/java/io/grpc/gcp/observability/logging/GcpLogSinkTest.java

```diff
@@ -98,32 +100,31 @@ public class GcpLogSinkTest {
       .putFields("rpc_id", Value.newBuilder().setStringValue(rpcId).build())
       .build();
   private Logging mockLogging;
+  private GcpLogSink spySink;
 
   @Before
   public void setUp() {
     mockLogging = mock(Logging.class);
+    spySink = spy(new GcpLogSink(destProjectName, locationTags,
+        customTags, flushLimit));
+    Mockito.doReturn(mockLogging).when(spySink).createLoggingClient();
   }
 
   @Test
   public void createSink() {
-    Sink mockSink = new GcpLogSink(mockLogging, destProjectName, locationTags,
-        customTags, flushLimit);
-    assertThat(mockSink).isInstanceOf(GcpLogSink.class);
+    assertThat(spySink).isInstanceOf(GcpLogSink.class);
   }
 
   @Test
   @SuppressWarnings("unchecked")
   public void verifyWrite() throws Exception {
-    Sink mockSink = new GcpLogSink(mockLogging, destProjectName, locationTags,
-        customTags, flushLimit);
-    mockSink.write(logProto);
+    spySink.write(logProto);
 
     ArgumentCaptor<Collection<LogEntry>> logEntrySetCaptor = ArgumentCaptor.forClass(
         (Class) Collection.class);
     verify(mockLogging, times(1)).write(logEntrySetCaptor.capture());
     for (Iterator<LogEntry> it = logEntrySetCaptor.getValue().iterator(); it.hasNext(); ) {
       LogEntry entry = it.next();
-      System.out.println(entry);
       assertThat(entry.getPayload().getData()).isEqualTo(expectedStructLogProto);
```

Comment:
Just noticed `expectedStructLogProto` - can it be defined in upper-case as `EXPECTED_STRUCT_LOG_PROTO` since it is like a constant object?

Reply:
similarly `logProto` ?

Reply:
Updated